### PR TITLE
add address field to sensors table

### PR DESCRIPTION
**Pull request by tmarkson**
a user on discourse had trouble so I thought this would be helpful.
https://meshtastic.discourse.group/t/ina219-telemetry-gotcha/6562

### CODE REVIEW

File: docs/configuration/module-config/telemetry.mdx [hunk 1]
```diff
@@ -16,16 +16,16 @@ The telemetry module config options are: Device Metrics Update Interval, Environ
...
-| INA260  | Current and Voltage                                           |
-| INA219  | Current and Voltage                                           |
-| LPS22   | Barometric pressure                                           |
-| SHTC3   | Temperature and humidity                                      |
+| Sensor  | I<sup>2</sup>C Address |  Data Points                                                  |
+| :-----: | :--------------------: | :-----------------------------------------------------------: |
+| BMP280  | 0x76                   | Temperature and barometric pressure                           |
+| BME280  | 0x76                   | Temperature, barometric pressure and humidity                 |
+| BME680  | 0x76                   | Temperature, barometric pressure, humidity and air resistance |
+| MCP9808 | 0x18                   | Temperature                                                   |
```
Comment (caveman99) [inline docs/configuration/module-config/telemetry.mdx:24]: All Bosch sensors will be detected on 0x76 AND 0x77

File: docs/configuration/module-config/telemetry.mdx [hunk 2]
```diff
@@ -16,16 +16,16 @@ The telemetry module config options are: Device Metrics Update Interval, Environ
...
+| INA260  | 0x40                   | Current and Voltage                                           |
+| INA219  | 0x40                   | Current and Voltage                                           |
```
Comment (caveman99) [inline docs/configuration/module-config/telemetry.mdx:26]: INA is on 0x40 and 0x41

File: docs/configuration/module-config/telemetry.mdx [hunk 3]
```diff
@@ -16,16 +16,16 @@ The telemetry module config options are: Device Metrics Update Interval, Environ
...
+| LPS22   | 0x5D                   | Barometric pressure                                           |
```
Comment (caveman99) [inline docs/configuration/module-config/telemetry.mdx:27]: LPS22 is detected on 0x5c and 0x5d

**Review by caveman99 — CHANGES_REQUESTED** [review]

**Review by tmarkson — COMMENTED** [review]
Updated the specific lines as requested by @caveman99

**Review by caveman99 — APPROVED** [review]

**Pull request merged**